lib/index.js

```diff
@@ -172,6 +172,7 @@ Segment.prototype.normalize = function(msg) {
   msg.userId = msg.userId || user.id();
   msg.anonymousId = user.anonymousId();
   msg.sentAt = new Date();
+  extend(msg, this._extras);
```

Comment:
I had thought of this as a generic "attach top-level properties" method, but it probably doesn't make sense. I'll make it specific to `__metadata__` (and rename the member variable)